build.gradle

```diff
@@ -259,6 +259,7 @@ dependencies {
     compile(group: 'io.searchbox', name: 'jest', version: '6.3.1') {
         exclude group: 'org.apache.logging.log4j', module: 'log4j-core'
     }
+    //RDM-8901 to bump version to ES7 ASAP
```

Comment:
can you add a similar comment on the jest library as well please? we can say:
`//RDM-8901 to bump version to an ES7 compatible one ASAP`